protocols/luci-proto-yggdrasil/htdocs/luci-static/resources/protocol/yggdrasil.js

```diff
@@ -0,0 +1,272 @@
+});
+
+var getPeers = rpc.declare({
+	object:'luci.yggdrasil',
+	method:'getPeers',
+	params:['interface'],
+	expect:{peers:[]}
+});
+
+return network.registerProtocol('yggdrasil',
+	{
+		getI18n: function() {
+			return _('Yggdrasil Network');
+		},
+		getIfname: function() {
+			return this._ubus('l3_device') || this.sid;
+		},
+		getType: function() {
+			return "tunnel";
+		},
+		getOpkgPackage: function() {
+			return 'yggdrasil';
+		},
+		isFloating: function() {
+			return true;
+		},
+		isVirtual: function() {
+			return true;
+		},
+		getDevices: function() {
+			return null;
+		},
+		containsDevice: function(ifname) {
+			return(network.getIfnameOf(ifname)==this.getIfname());
+		},
+		renderFormOptions: function(s) {
+			var o, ss;
+			o=s.taboption('general',form.Value,'private_key',_('Private key'),_('The private key for your Yggdrasil node'));
+			o.optional=false;
+			o.password=true;
+			o.validate=validatePrivateKey;
+
+			o=s.taboption('general',form.Value,'public_key',_('Public key'),_('The public key for your Yggdrasil node'));
+			o.optional=false;
+			o.validate=validatePublicKey;
+
+			s.taboption('general',cbiKeyPairGenerate,'_gen_server_keypair',' ');
+
+			o=s.taboption('advanced',form.Value,'mtu',_('MTU'),_('Specify an MTU (Maximum Transmission Unit) for your local TUN interface. Default is the largest supported size for your platform. The lowest possible value is 1280.'));
+			o.optional=true;
+			o.placeholder=1280;
+			o.datatype='range(1280, 65535)';
+
+			o=s.taboption('general',form.TextValue,'node_info',_('Node info'),_('Optional node info. This must be a { "key": "value", ... } map or set as null. This is entirely optional but, if set, is visible to the whole network on request.'));
+			o.optional=true;
+			o.placeholder="{}";
+
+			o=s.taboption('general',form.Flag,'node_info_privacy',_('Node info privacy'),_('By default, node info contains some defaults including the platform, architecture and Yggdrasil version. These can help when surveying the network and diagnosing network routing problems. Enabling node info privacy prevents this, so that only items specified in "Node info" are sent back if specified.'));
+			o.default=o.disabled;
+
+			try {
+				s.tab('peers',_('Peers'));
+			} catch(e) {};
+			o=s.taboption('peers', form.SectionValue, '_active', form.NamedSection, this.sid, "interface", _("Active peers"))
+			ss=o.subsection;
+			ss.option(cbiActivePeers, this.sid);
+
+			o=s.taboption('peers', form.SectionValue, '_listen', form.NamedSection, this.sid, "interface", _("Listen for peers"))
+			ss=o.subsection;
+
+			o=ss.option(form.DynamicList,'listen_address',_('Listen addresses'),_('Listen addresses for incoming connections. You will need to add listeners in order to accept incoming peerings from non-local nodes. Multicast peer discovery will work regardless of any listeners set here. Each listener should be specified in URI format, e.g.tls://0.0.0.0:0 or tls://[::]:0 to listen on all interfaces.'));
```

Comment:
`_('Add listeners in order to accept incoming peerings from non-local nodes. Multicast peer discovery works regardless of listeners set here. URI Format: <code>tls://0.0.0.0:0</code> or <code>tls://[::]:0<code> to listen on all interfaces.')`

Reply:
- added the wording and added compatible URIs 
![image](https://github.com/openwrt/luci/assets/5460439/8636ea7e-6f9f-4691-b085-97d4f44490d0)